apps/src/codebridge/ControlButtons/index.tsx

```diff
@@ -111,7 +114,7 @@ const ControlButtons: React.FunctionComponent = () => {
 
   // We disabled navigation if we are still loading, or if this is a submittable level,
   // the user has not submitted yet, and the user has not run their code during this session.
-  const disableNavigation = isSubmittable && !hasSubmitted && !hasRun;
+  const awaitingSubmitRun = isSubmittable && !hasSubmitted && !hasRun;
```

Comment:
nit: `awaitingSubmitorRun`?

Reply:
This is actually "are we a submit level and we have not yet run code", so `or` is not relevant here. I'm going to be getting rid of this variable in a follow-up PR where I update the logic for enabling/disabling the continue/finish/submit button so I think I'll leave this kind of confusing name for now.